test/unit/storage/index.spec.ts

```diff
@@ -2,14 +2,14 @@
  * @license
  * Copyright 2021 Google Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
This and below changes in `"` -> `'` are probably from the auto fix in the linter. The rest of the codebase use `"`. Let's keep it consistence in this file as well. If we want to change it we should change everywhere and in a separate PR.